include/cute/atom/copy_atom.hpp

```diff
@@ -111,6 +111,10 @@ struct Copy_Atom<Copy_Traits<Args...>, CopyInternalType>
       //   recurse this rank-1 layout by peeling off the mode
       //   ((A,B,C,...)) -> (A,B,C,...)
       return copy(*this, tensor<0>(src), tensor<0>(dst));
+    } else if constexpr (is_tuple<typename Tensor<SEngine,
+                                 SLayout>::engine_type::iterator::
+                                         value_type>::value) {
+        return copy_unpack(*this, src, dst);
```

Comment:
As discussed please remove this elseif condition and make the changes in the copy traits definition so the execution go into the first if check

Reply:
For this you need to make `SrcLayout` in the prefetch copy trait to match `decltype(size(src))>::value`